src/client/pages/CubeBlogPage.tsx

```diff
@@ -33,7 +33,7 @@ const CubeBlogPage: React.FC<CubeBlogPageProps> = ({ cube, lastKey, posts, login
 
   return (
     <MainLayout loginCallback={loginCallback}>
-      <CubeLayout cube={cube} activeLink="blog" hasControls={user != null && cube.owner.id == user.id}>
```

Comment:
This form `user != null` worked before and after, so its more of a style change for consistency across pages. In most user was checked like `user ? A : B` or `!!user { .. }`